src/pbt/v2/deployment/pipeline.py

```diff
@@ -253,13 +259,11 @@ def mvn_build(self):
     def wheel_build(self):
         response_code = 0
         if self._are_tests_enabled:
-            test_command = ["python3", "-m", "pytest", "-v", f"{self._base_path}/test/TestSuite.py",
-                            f"--html={self._base_path}/report.html",
-                            "--self-contained-html", f"--junitxml={self._base_path}/report.xml"""]
+            test_command = ["python3", "-m", "pytest", "-v", f"{self._base_path}/test/TestSuite.py"]
             log(f"Running python test {test_command}", step_id=self._pipeline_id)
             response_code = self._build(test_command)
 
-            if response_code != 0:
+            if response_code not in (0, 5):
```

Comment:
what is response code 5? Maybe put a comment here?

Reply:
I have. updated that in description. 
Will update here as well.